packages/gafl-webapp-service/src/pages/multibuy/view-licences/route.js

```diff
@@ -1,29 +1,45 @@
 import pageRoute from '../../../routes/page-route.js'
 import Joi from 'joi'
-import { VIEW_LICENCES } from '../../../uri.js'
+import { LICENCE_FOR, VIEW_LICENCES } from '../../../uri.js'
 import { licenceTypeDisplay, licenceTypeAndLengthDisplay } from '../../../processors/licence-type-display.js'
 import { displayStartTime } from '../../../processors/date-and-time-display.js'
 import { nextPage } from '../../../routes/next-page.js'
 import { hasDuplicates } from '../../../processors/multibuy-processor.js'
+import { addLanguageCodeToUri } from '../../../processors/uri-helper.js'
 
 export const getData = async request => {
   const transaction = await request.cache().helpers.transaction.get()
   const mssgs = request.i18n.getCatalog()
+  const licencesRemaining = transaction.permissions[0].licensee.firstName !== undefined
 
-  const licences = transaction.permissions.map((permission, index) => ({
-    licenceHolder: `${permission.licensee.firstName} ${permission.licensee.lastName}`,
-    type: licenceTypeDisplay(permission, mssgs),
-    length: licenceTypeAndLengthDisplay(permission, mssgs),
-    start: displayStartTime(request, permission),
-    price: permission.permit.cost,
-    index
-  }))
+  if (licencesRemaining) {
+    const licences = transaction.permissions.map((permission, index) => ({
+      licenceHolder: `${permission.licensee.firstName} ${permission.licensee.lastName}`,
+      type: licenceTypeDisplay(permission, mssgs),
+      length: licenceTypeAndLengthDisplay(permission, mssgs),
+      start: displayStartTime(request, permission),
+      price: permission.permit.cost,
+      index
+    }))
 
-  const duplicate = await hasDuplicates(licences)
+    const duplicate = await hasDuplicates(licences)
 
+    return {
+      duplicate,
+      licences,
+      licencesRemaining,
+      uri: {
+        licence_for: addLanguageCodeToUri(request, LICENCE_FOR.uri)
+      }
+    }
+  }
   return {
-    duplicate,
-    licences
+    duplicate: false,
+    licences: undefined,
+    licencesRemaining,
+    uri: {
+      licence_for: addLanguageCodeToUri(request, LICENCE_FOR.uri)
+    }
```

Comment:
What's this second return statement for? It should be deleted, the code can never reach it due to the preceding return statement.

Reply:
for when licences remaining isn't true

Reply:
Ah, my mistake, sorry!